module/commands/regolamento_didattico.py

```diff
@@ -73,10 +88,14 @@ def send_regolamento(update: Update, context: CallbackContext):
     query = update.callback_query
     data = query.data
     chat_id = update.effective_chat.id
-    if data in reg_doc_triennale:
-        doc = reg_doc_triennale[data]
+    if data in reg_doc_triennale_L31:
+        doc = reg_doc_triennale_L31[data]
+    if data in reg_doc_triennale_L35:
+        doc = reg_doc_triennale_L35[data]
+    if data in reg_doc_magistrale_LM18:
+        doc = reg_doc_magistrale_LM18[data]
     else:
-        doc = reg_doc_magistrale[data]
+        doc = reg_doc_magistrale_LM40[data]
```

Comment:
in python `switch` does not exist, so use `if / elif /else` instead of a lot of `if`